res/i18n/fr/labels.json

```diff
@@ -481,38 +481,38 @@
 		"enabled": "Activé",
 		"fade_in": "Fondu ouverture",
 		"fade_out": "Fondu fermeture",
-		"head_cut": "Écrêtage",
+		"head_cut": "Coupe du début",
 		"listen": "Écoute",
-		"makeup": "Surgain",
+		"makeup": "Gain",
 		"predelay": "Pré-délai",
 		"reverse": "Inverser",
 		"sample_num": "Échantillon #",
-		"tail_cut": "Équeutage"
+		"tail_cut": "Coupe de la fin"
 	},
 	"settings": "Paramètres",
 	
-	"side_direction": "Direction de côté", 
+	"side_direction": "Cotés directivité", 
 	"sidechain": "Chaîne latérale",
 	
 	"signal": {
 		"M_iddle": "M",
 		"P_hase": "P",
 		"S_ide": "C",
 		"amplitude": "Amplitude",
-		"clip": "Clip",
-		"clipping": "Clipping",
+		"clip": "Saturation",
+		"clipping": "Saturation",
 		"dry": "Originel",
 		"dry:db": "Originel\n(dB)",
 		"dry_pan": "Pan originel",
-		"duty_ratio": "Ratio obligatoire",
+		"duty_ratio": "Rapport cyclique",
 		"envelope": "Enveloppe",
 		"fade_in": "Fondu ouverture",
 		"fade_out": "Fondu fermeture",
 		"fall": "Chute",
 		"in_lr": "Entrée G/D",
 		"invert": "Inverser",
 		"mono": "Mono",
-		"negative_width": "Largeur négative",
+		"negative_width": "Largeur négatif",
```

Comment:
"largeur" is a feminine noun, hence the adjective needs to match it.
Please revert.

Reply:
ping @wargreen